### ISSUE TITLE
Criar README

### ISSUE THREAD

**Issue by jamesperes-zz** [samukasmk/bora_falar_de_python#3]

**Comment by jamesperes-zz** [comment]
@Riverfount o arquivo já foi criado

**Comment by Riverfount** [comment]
@jamesperes foi criado, mas só tem uma frase eheheh tem que implementar algo mais interessante, inclusive com formas de como forkar e iniciar os trabalhos para aqueles que vieram a ajudar nos códigos!!!

**Comment by jamesperes-zz** [comment]
como eu vi que vc se atribuiu pensei que fosse fazer, por isso que dei um toque,
dá uma olhada nisso, pode te ajudar bastante
https://gist.github.com/PurpleBooth/109311bb0361f32d87a2

**Comment by Riverfount** [comment]
@jamesperes sim, marquei-me justamente para implementá-lo. Grato pela dica! Será muito útil mesmo, gostei muito e já dei uma estrela lá heheheh.... espero até início do ano que vem deixar pronto esse READEME.md daqui.... Boas festas e um 2018 muito produtivo a todos nós!
